client/state/data-layer/wpcom/sites/rewind/schema.js

```diff
@@ -0,0 +1,102 @@
+/** @format */
+
+export const credential = {
+	type: 'object',
+	properties: {
+		still_valid: { type: 'boolean' },
+		type: { type: 'string', enum: [ 'auto', 'ftp', 'sftp', 'ssh' ] },
+		host: { type: 'string' },
+		port: { type: 'integer' },
+	},
+	required: [ 'still_valid', 'type' ],
+};
+
+export const download = {
+	type: 'object',
+	properties: {
+		downloadId: { type: 'integer' },
+		rewindId: { type: 'string' },
```

Comment:
I am guessing we do accept an string here. Just want to make sure we didn't mean integer.

Reply:
it was originally an integer I think, then a float, then a float represented as a definite string 🤦‍♂️